src/AutoRest.CSharp/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -86,14 +100,14 @@ public async Task<bool> Execute(IPluginCommunication autoRest)
             string csProjContent;
             if (configuration.SkipCSProjPackageReference)
```

Comment:
We should eventually cleanup this file to have a bit less duplication. It seems most differences are in depednecies.